app/services/clients/connector_client.js

```diff
@@ -53,6 +54,9 @@ const _getDeleteTokenUrlFor = tokenId => baseUrl + CARD_DELETE_CHARGE_TOKEN_PATH
 /** @private */
 const _getPatchUrlFor = chargeId => baseUrl + CARD_CHARGE_PATH.replace('{chargeId}', chargeId)
 
+/** @private */
```

Comment:
Does this line do anything?

Reply:
i doubt it, i don't really know. doesn't look like we use this notation anywhere else - not sure why it's done here

Reply:
ask @jonheslop !

Reply:
he doesn't know. i may remove it all in another pr if i get the chance